src/hotspot/share/runtime/thread.cpp

```diff
@@ -471,6 +471,7 @@ void Thread::print_on(outputStream* st, bool print_extended_info) const {
     if (!is_Java_thread() || !JavaThread::cast(this)->is_vthread_mounted()) {
       osthread()->print_on(st);
     }
+    st->print("stack=[" PTR_FORMAT ", " PTR_FORMAT ")", p2i(stack_end()), p2i(stack_base()));
```

Comment:
We already report the thread stack elsewhere, so is this going to duplicate the information?